### PR TITLE
HPCC-22003 Implement Portuguese Doc Build

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>
@JamesDeFabia please review
NOTE: This is same as PR # 12469 only targets candidate-7.0.x

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Successful Build Test: http://10.240.32.243/view/Docs%207.0/job/PT_BR_Build1/

### CODE REVIEW

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
I reviewed the output and they look great! 
Only one minor thing, the cmake.txt files under PT_BR are new and the copyright line has various dates such as:
HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®
I think they should all say 2019

**Comment by g-pan** [comment]
@JamesDeFabia back to you

File: docs/PT_BR/ECLScheduler/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿################################################################################
-#    HPCC SYSTEMS software Copyright (C) 2012 HPCC SystemsÂ®.
+#    HPCC SYSTEMS software Copyright (C) 2018 HPCC Systems®.
```
Comment (JamesDeFabia) [inline docs/PT_BR/ECLScheduler/CMakeLists.txt]: Why 2018?

File: docs/PT_BR/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,105 @@
+﻿################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2018 HPCC Systems®.
```
Comment (JamesDeFabia) [inline docs/PT_BR/CMakeLists.txt]: Why 2018?

File: docs/PT_BR/HPCCCertify/CMakeLists.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ﻿################################################################################
-#    HPCC SYSTEMS software Copyright (C) 2012 HPCC SystemsÂ®.
+#    HPCC SYSTEMS software Copyright (C) 2018 HPCC Systems®.
```
Comment (JamesDeFabia) [inline docs/PT_BR/HPCCCertify/CMakeLists.txt]: Why 2018?

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Still some (c) date inconsistencies

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
@g-pan two dates missed

**Comment by g-pan** [comment]
@JamesDeFabia good to merge?

**Review by JamesDeFabia — APPROVED** [review]
@richardkchapman this is now good to merge

**Pull request merged**